checker/raw_result.go

```diff
@@ -125,10 +125,35 @@ type MaintainedData struct {
 	ArchivedStatus       ArchivedStatus
 }
 
+type LicenseAttributionType string
+
+const (
+	// sources of license information used to assert repo's license.
+	LicenseAttributionTypeOther     LicenseAttributionType = "otherAttribution"
+	LicenseAttributionTypeRepo      LicenseAttributionType = "repoAttribution"
```

Comment:
Can we make these more self-explanatory? Like `API`, `heuristics`. I'm sure we can find better than what I'm suggesting, though. Thoughts?

Reply:
Two thoughts... I'd like your opinion:

1. see the new "checks" documentation, where it states that License using either a repo API (so I like `API`) in your suggestion/comment, and goes on to state "_Scorecard attempts to detect a published license file on its own_"-- to use `heuristics`, then (which I also like), I'd change the documentation to read something like "_Scorecard uses its own heuristics to detect a
published license file._"

2. `checks/evaluation/license.go` is the only reference to `checker.LicenseAttributionTypeOther` and since it is in a switch/case statement, I could remove it and eliminate the const, but I think the evaluation code is better off than falling into `default` in the event other const's are established later -- your call!

Reply:
For users who want to use the raw results to automate parsing the results, I was hoping the name could be more explanatory in the field of the results as well. Something like `repositoryAPI` and `builtinHeuristics` could work.

Doc + evaluation code LGTM

Reply:
I'd go with this for the code overall:

```
const (
        // sources of license information used to assert repo's license.
        LicenseAttributionTypeOther      LicenseAttributionType = "other"
        LicenseAttributionTypeAPI        LicenseAttributionType = "repositoryAPI"
        LicenseAttributionTypeHeuristics LicenseAttributionType = "builtinHeuristics"
)
```